src/test/regress/sql/create_index.sql

```diff
@@ -975,3 +975,89 @@ RESET enable_indexonlyscan;
 
 explain (costs off)
   select * from tenk1 where (thousand, tenthous) in ((1,1001), (null,null));
+
+--
+-- Check spgist index work with AO tables
+--
+-- insert to AO table with spgist index
+create table spgist_ao_test(idx_col int4range)
```

Comment:
In this file all other SQL words are in capital (**CREATE** but not **create**).
Should not it be consistent with the rest of the file?

Reply:
I don't think it make real sense here. There are no strict rules explaining this format, and more, this set of tests just continues existing vanilla Postgres [test](https://github.com/arenadata/gpdb/pull/241/files/e9dc8c11668fd43e94290573ed6d102ce21d6b3f#diff-6a13a6d37140c67217997c422b6a960bbb089f280bd8c43b0bc6c339a88b606dR976) above, which is all in lowercase.